packages/api/src/hooks/useP2PAdvertList.ts

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import useInfiniteQuery from '../useInfiniteQuery';
+import useAuthorize from './useAuthorize';
+
+/**
+ * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
+ */
+const useP2PAdvertList = (
```

Comment:
I think its better to separate p2p hooks in its own folder from now.

Reply:
agree with @thisyahlen-deriv. @adrienne-deriv can you also please check the code smells.